### PR TITLE
ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example

### PR DESCRIPTION
**Pull request by KazeemHamzat**

### CODE REVIEW

File: ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  "requester": {
+    "reference": "PractitionerRole/PractitionerRole-GeneSmithENT-Example"
+  },
+  "reasonCode": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/England-GenomicTestDirectory",
+          "code": "R387",
+          "display": "Re-analysis"
```
Comment (omarisgreat) [inline ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json:58]: Here, might need to look at either using a SNOMED code (such as 719641000000101 | Data analysis (qualifier value) |) or proposing an addition to SNOMED (as this is how diagnostic/carrier have been coded in previous examples)

Comment (omarisgreat) [reply ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json:58]: Could also add a text element here "Significant new treatment and or clinical management implication for the patient, treatment for management of epilepsy" (as this is the reason specified for requesting reanalysis)

File: ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
+{
+  "resourceType": "ServiceRequest",
```
Comment (omarisgreat) [inline ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json:2]: Should add a basedOn field as this is a reanalysis that is related to a previous order (can add a fictitious ServiceRequest ID reference

File: ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-FundingCategory",
+            "code": "nhs",
+            "display": "NHS"
+          }
+        ]
+      }
+    }
+  ],
+  "status": "active",
+  "intent": "reflex-order",
```
Comment (omarisgreat) [inline ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json:19]: I think this would be classed as an order rather than a reflex as this is a new request rather than being requested based on the results of a previous test

File: ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      "reference": "FamilyMemberHistory/Observation-NonConsanguinousUnion-Example"
+    },
+    {
+      "reference": "Observation/Observation-NoBoneMarrowTransplant-Example"
+    },
+    {
+      "reference": "Observation/Observation-NoTransfusion-Example"
+    },
+    {
+      "reference": "DiagnosticReport/UKCore-DiagnosticReport-MichaelJonesReport-Example"
```
Comment (omarisgreat) [inline ServiceRequest/ServiceRequest-ReAnalysis-NonWGSTestOrderForm-Example.json:83]: Can remove this as not previous test reports have actually been attached in the example

File: ServiceRequest/ServiceRequest-NonWGSTestOrderForm-Example.json [hunk 5]
```diff
@@ -79,6 +79,9 @@
 		{
 			"reference": "Observation/Observation-NoTransfusion-Example"
 		},
+		{
+			"reference": "Observation/Observation-NoBoneMarrowTransplant-Example"
```
Comment (omarisgreat) [inline ServiceRequest/ServiceRequest-NonWGSTestOrderForm-Example.json:83]: Was this an omission on the previous Non-WGS example? If so, could you split this out into a separate PR?

**Review by omarisgreat — CHANGES_REQUESTED** [review]

File: ServiceRequest/ServiceRequest-NonWGSTestOrderForm-Example.json (hunk 5 above)
Comment (KazeemHamzat) [reply ServiceRequest/ServiceRequest-NonWGSTestOrderForm-Example.json:83]: Yes

File: ServiceRequest/ServiceRequest-NonWGSTestOrderForm-ReAnalysis-Example [hunk 6]
```diff
@@ -71,24 +85,29 @@
...
     },
     {
       "reference": "Observation/Observation-NoTransfusion-Example"
-    },
+    }
+  ],
+  "specimen": [
     {
-      "reference": "DiagnosticReport/UKCore-DiagnosticReport-MichaelJonesReport-Example"
+      "display": "No Sample Required"
```
Comment (omarisgreat) [inline ServiceRequest/ServiceRequest-NonWGSTestOrderForm-ReAnalysis-Example:102]: Could you remove the specimen element and add this text to note instead

File: Task/Task-NonWGSRareDiseaseTestOrderAccepted-Reanlaysis-Example.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
+{
+    "resourceType": "Task",
+    "id": "Task-NonWGSRareDiseaseTestOrderAccepted-Reanlaysis-Example",
+    "basedOn": [
+        {
+            "reference": "ServiceRequest/ServiceRequest-NonWGSTestOrderForm-ReAnalysis-Example.json"
```
Comment (omarisgreat) [inline Task/Task-NonWGSRareDiseaseTestOrderAccepted-Reanlaysis-Example.json:6]: remove basedOn

File: Task/Task-NonWGSRareDiseaseTestOrderAccepted-Reanlaysis-Example.json [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        }
+    ],
+    "status": "accepted",
+    "intent": "order",
+    "priority": "routine",
+    "code": {
+        "coding": [
+            {
+                "system": "https://fhir.nhs.uk/CodeSystem/task-code-genomics",
+                "code": "RequestGenomicTest",
```
Comment (omarisgreat) [inline Task/Task-NonWGSRareDiseaseTestOrderAccepted-Reanlaysis-Example.json:16]: change code to Process Genomic Test Request

**Review by omarisgreat — APPROVED** [review]
A few small changes to the Task/SR needed but happy for it to be merged in after these are done without requiring another review
